dbt/adapters/trino/connections.py

```diff
@@ -408,7 +409,7 @@ def open(cls, connection):
         trino_conn = trino.dbapi.connect(
             host=credentials.host,
             port=credentials.port,
-            user=credentials.user,
+            user=credentials.impersonate_user or credentials.user,
```

Comment:
pls add a corresponding test in `test_adapter.py`

Reply:
Sure, i am not sure where to assert the dbapi user. Can you point me to it?